src/ariel-os-esp/src/lib.rs

```diff
@@ -86,7 +86,7 @@ pub mod peripherals {
             pub type GPIO29 = esp_hal::gpio::GpioPin<29>;
             pub type GPIO30 = esp_hal::gpio::GpioPin<30>;
         }
-        else if #[cfg(context = "esp32")] {
+        else if #[cfg(any(context = "esp32", context = "esp32s3"))] {
```

Comment:
Good catch! We'll fix the pin list in a separate PR for the ESP32-S3.

```suggestion
        else if #[cfg(context = "esp32")] {
```